### PR TITLE
Add CanActivate Guard and SingInService

### PR DESCRIPTION
**Pull request by DimaUstim**

### CODE REVIEW

File: src/app/service/sing-in/auth-guard.service.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+@Injectable()
+export class AuthGuardService implements CanActivate {
+    constructor(private singInService: SingInService, private router: Router) { };
+
+    canActivate(route: ActivatedRouteSnapshot,
+        state: RouterStateSnapshot): boolean {
+        let isLoggedIn = this.singInService.isUserLoggedIn;
+        if (isLoggedIn) {
+
```
Comment (AndriiPanasenkoSolvve) [inline src/app/service/sing-in/auth-guard.service.ts]: Remove unnecessary empty lines please

File: src/app/service/sing-in/auth-guard.service.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            return true;
+
+        } else {
```
Comment (AndriiPanasenkoSolvve) [inline src/app/service/sing-in/auth-guard.service.ts]: As here is no some special logic, you can just use

> return isLoggedIn

**Review by AndriiPanasenkoSolvve — CHANGES_REQUESTED** [review]
Remove unnecessary code formatting changes from the request please

File: src/app/service/sing-in/auth-guard.service.ts (hunk 1 above)
Comment (DimaUstim) [reply src/app/service/sing-in/auth-guard.service.ts]: Fixed

File: src/app/service/sing-in/auth-guard.service.ts (hunk 2 above)
Comment (DimaUstim) [reply src/app/service/sing-in/auth-guard.service.ts]: Fixed

**Pull request closed without merging**